### PR TITLE
Develop mike sanin

### PR DESCRIPTION
**Pull request by mikesanin**
mike_sanin comment ?

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
-
-    :param phrase: phrase to count in it
-    :param letter: letter to find occurrences of it
-    :return: count occurrences of letter in phrase
-    """
-    # write your code here
+sorted_variables = {
+    "mutable": [b, d],
+    "immutable": [a, c]
+}
```
Comment (EdAlekseiev) [inline app/main.py:9]: Incorrect solution. Please, re-read the task requirements.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — CHANGES_REQUESTED** [review]
The code is good, but try to remove pycache files from git repo. They are not needed.

File: app/main.py [hunk 2]
```diff
@@ -1,17 +1,14 @@
-def count_occurrences(phrase: str, letter: str) -> int:
-    """
-    Implement count_occurrences function:
+variable_one = 123
+empty_list = []
+greeting = "Hi!"
+number_list = [1, 2]
```
Comment (Bohdan-Salamakha) [inline app/main.py:4]: remove this redundant code

File: app/main.py [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
 
-    :param phrase: phrase to count in it
-    :param letter: letter to find occurrences of it
-    :return: count occurrences of letter in phrase
-    """
-    # write your code here
+    return count
```
Comment (Bohdan-Salamakha) [inline app/main.py:14]: https://www.w3schools.com/python/ref_string_count.asp

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
remove all pycache files/folders from github

**Pull request closed without merging**